src/test/java/com/exasol/closeafterall/CloseAfterAllExtension.java

```diff
@@ -1,16 +1,17 @@
 package com.exasol.closeafterall;
 
-import com.exasol.errorreporting.ExaError;
-import org.junit.jupiter.api.extension.AfterAllCallback;
-import org.junit.jupiter.api.extension.ExtensionContext;
-
 import java.io.Closeable;
 import java.io.IOException;
 import java.lang.reflect.Field;
 
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import com.exasol.errorreporting.ExaError;
+
 /**
  * This JUnit extension closes resources in static fields that are annotated with {@code @CloseAfterAll}. You can use it
- * by annotating your class with {@code @ExtendWith({ CloseAfterAllExtension.class })}.
+ * by annotating your class with {@code @ExtendWith(&#123; CloseAfterAllExtension.class &#125;)}.
```

Comment:
Wow number two: This also sounds helpful fixing warnings about potentially unclosed resources, right?